### PR TITLE
utils: fix unicode issues in utils/conferences

### LINKED ISSUE #1826: utils/conferences.py: does not handle correctly unicode
From QA:
```
2016-12-18 03:39:01,908 ERROR: Exception on /ajax/conferences/contributions [GET] [in /opt/inspire/lib/python2.7/site-packages/flask/app.py:1587]
Traceback (most recent call last):
  File "/opt/inspire/lib/python2.7/site-packages/flask/app.py", line 1988, in wsgi_app
    response = self.full_dispatch_request()
  File "/opt/inspire/lib/python2.7/site-packages/flask/app.py", line 1641, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/opt/inspire/lib/python2.7/site-packages/flask/app.py", line 1544, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/opt/inspire/lib/python2.7/site-packages/flask/app.py", line 1639, in full_dispatch_request
    rv = self.dispatch_request()
  File "/opt/inspire/lib/python2.7/site-packages/flask/app.py", line 1625, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/opt/inspire/src/inspire/inspirehep/modules/theme/views.py", line 610, in ajax_conference_contributions
    html, total = render_conferences_contributions(cnum)
  File "/opt/inspire/src/inspire/inspirehep/utils/conferences.py", line 51, in render_conferences_contributions
    return render_contributions(conferences_contributions_from_es(cnum))
  File "/opt/inspire/src/inspire/inspirehep/utils/conferences.py", line 128, in render_contributions
    name=get_title(hit.to_dict())
UnicodeEncodeError: 'ascii' codec can't encode character u'\u2014' in position 38: ordinal not in range(128)
```

Haven't checked how to trigger it though.

### PR DESCRIPTION
**Pull request by jacquerie**
Sentry: https://sentry.cern.ch/inspire-sentry/inspire-labs-qa/group/606817/

Closes #1826

### CODE REVIEW

File: tests/unit/utils/test_utils_conferences.py [hunk 1]
```diff
@@ -120,3 +120,34 @@ def test_render_contributions():
...
+                },
+            ],
+            'total': 1,
+        },
+    }).hits
+
+    expected = ([
+        [
+            u"<a href='/literature/1427573'>Storage Ring Based EDM Search — Achievements and Goals</a>",
+            u'\n  \n  \n\n\n  \n',
```
Comment (kaplun) [inline tests/unit/utils/test_utils_conferences.py]: Please, make this test to ignore whitespaces. In this there won't be a need to amend it each time someone re-indent the template.

Comment (jacquerie) [inline tests/unit/utils/test_utils_conferences.py]: You can't. That's the output of the function. It's the code that is brittle, not the test.

**Comment by kaplun** [comment]
Can you fix the identical bug at https://github.com/inspirehep/inspire-next/pull/1855/files#diff-10890929d64b89c555a10128b8f09dedL101?

**Comment by jacquerie** [comment]
> Can you fix the identical bug at https://github.com/inspirehep/inspire-next/pull/1855/files#diff-10890929d64b89c555a10128b8f09dedL101?

When I need non-ascii values I always turn to France: http://inspire-nightly.cern.ch/conferences/1351301

File: tests/unit/utils/test_utils_conferences.py (hunk 1 above)
Comment (jacquerie) [inline tests/unit/utils/test_utils_conferences.py]: My point is, there's no point sweeping some ugly output under the rug to pretend it doesn't exist, but we should take it as an occasion to ["listen to our tests"](http://chimera.labs.oreilly.com/books/1234000000754/ch19.html#_listen_to_your_tests_ugly_tests_signal_a_need_to_refactor): this "API" function that returns formatted HTML is bad, and should be refactored.

Besides, a fix for this is already coming in https://github.com/inspirehep/inspire-next/pull/1785/commits/0b90eac5baa560c34b6ece0cef23dbdb2ca6ddce.

**Pull request merged**